ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { PRIORITY_LEVELS } from '../../../../../shared/constants/gas';
+import {
+  COLORS,
+  FONT_WEIGHT,
+} from '../../../../helpers/constants/design-system';
+import Typography from '../../../ui/typography';
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+
+const EditGasToolTip = ({
+  priorityLevel,
+  // maxFeePerGas & maxPriorityFeePerGas are derived from conditional logic
+  // related to the source of the estimates. We pass these values from the
+  // the parent component (edit-gas-item) rather than recalculate them
+  maxFeePerGas,
+  maxPriorityFeePerGas,
+  t,
+}) => {
+  const {
+    gasLimit,
+    maxFeePerGas: maxFeePerGasValue,
+    maxPriorityFeePerGas: maxPriorityFeePerGasValue,
```

Comment:
Despite the comment above, it seems that we are still recalculating them, so what is the purpose of also passing them down? Sorry if I'm being dense.

Reply:
So here were getting the `maxFeePerGas` and `maxPriorityFeePerGas` from the gasContext and renaming them to `maxFeePerGasValue` and `maxPriorityFeePerGasValue`  to use them as fallback values to the passed down values that are derived from logic on lines 45-67 in `edit-gas-item.js`

Reply:
Ah, ok, gotcha.